tests/CMakeLists.txt

```diff
@@ -1,15 +1,23 @@
 set(TESTS_SRC
-    main.cpp
-    test_morphology.cpp
-    test_mut_morphology.cpp
-)
+        main.cpp
+        test_morphology_readers.cpp
+        test_immutable_morphology.cpp
+        test_mutable_morphology.cpp
+        )
 set(TESTS_LINK_LIBRAIRIES morphio_static HighFive)
 
+if(APPLE)
+  add_definitions("-DLIBCXX_INSTALL_FILESYSTEM_LIBRARY=YES")
+  set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -stdlib=libc++")
+else()
+  set(TESTS_LINK_LIBRAIRIES ${TESTS_LINK_LIBRAIRIES} PUBLIC stdc++fs)
+endif()
+
 add_executable(unittests ${TESTS_SRC})
 
 set_target_properties(unittests
   PROPERTIES
-  CXX_STANDARD 11
+  CXX_STANDARD 17
```

Comment:
please add a comment saying that 17 is only for tests, and it's for access to std::filesystem, in particular

Reply:
done